lib/heartSvgs.ts

```diff
@@ -0,0 +1,194 @@
+export const selectMessageHeartSvg = `
+    <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 162 162" xml:space="preserve">
+    <g transform="translate(81,81)">
+        <path d="M79.2-43C71.2-78.4,30.8-84.9,5-60.9c-2.5,2.3-6.4,2.1-8.8-0.3c-25-25.9-75.1-15-76.7,28.2C-82.6,22.3-14,75.2,1.5,75.1C17.3,75.1,91.3,10.7,79.2-43z" fill="none" stroke="black" stroke-dasharray="8" stroke-width="4px"/>
+        <text style="fill:#FA0F95; text-anchor:middle; font-size:30px; font-family: monospace, monospace; font-weight: 500; letter-spacing: 2px;"><tspan x="0" y="-15">SELECT</tspan><tspan x="0" y="20">MSG</tspan></text>
+    </g>
+    </svg>
+    `
+
+export const sendToHeartSVG = `
+    <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 162 162" xml:space="preserve">
+    <g transform="translate(81,81)">
+        <path d="M79.2-43C71.2-78.4,30.8-84.9,5-60.9c-2.5,2.3-6.4,2.1-8.8-0.3c-25-25.9-75.1-15-76.7,28.2C-82.6,22.3-14,75.2,1.5,75.1C17.3,75.1,91.3,10.7,79.2-43z" fill="none" stroke="black" stroke-dasharray="8" stroke-width="4px"/>
+        <text style="fill:#FA0F95; text-anchor:middle; font-size:30px; font-family: monospace, monospace; font-weight: 500; letter-spacing: 2px;"><tspan x="0" y="-15">SEND</tspan><tspan x="0" y="20">TO?</tspan></text>
+    </g>
+    </svg>
+    `
+
+export const addrHeartSVG = (address: string) => {
+  return `
+    <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 162 162" xml:space="preserve">
+    <g transform="translate(81,81)">
+        <path d="M79.2-43C71.2-78.4,30.8-84.9,5-60.9c-2.5,2.3-6.4,2.1-8.8-0.3c-25-25.9-75.1-15-76.7,28.2C-82.6,22.3-14,75.2,1.5,75.1C17.3,75.1,91.3,10.7,79.2-43z" fill="#fff"/>
+        <text style="fill:#FA0F95; text-anchor:middle; font-size:30px; font-family: monospace, monospace; font-weight: 500; letter-spacing: 2px;"><tspan x="0" y="-10">${address}</tspan></text>
+    </g></svg>
+    
+    `
+}
+
+export function getTextHeartSVG(heartType: number) {
```

Comment:
@JFrankfurt are we ok with this using mine, or should I refactor to use your hearts? 😅